bin/setup_venv.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Use `#!/usr/bin/env bash` for portability (http://www.cyberciti.biz/tips/finding-bash-perl-python-portably-using-env.html).